packages/components/src/FormField/FormField.css

```diff
@@ -269,7 +269,7 @@
 .postfix {
   position: absolute;
   top: 50%;
-  right: var(--space-base);
```

Comment:
@taylorvnoj does this need to account for the formfield's `size` prop?

Reply:
@chris-at-jobber  Screenshot shows current on top and this PR on the bottom.  Currently the gap on the right is the same no matter the `FormField` size.  Is that something we want to change?

![Screenshot 2024-04-12 at 3 34 26 PM](https://github.com/GetJobber/atlantis/assets/104797704/42355e37-b3ee-4bb3-84ba-ff3e85007e19)

Reply:
Wait nvm, the figma has different spacing between the right edge and the caret so I'll see what I can do

Reply:
Alright, the Select > Sizes story shows the changes best and aligns with the[ designs](https://www.figma.com/file/rIIhulZvcp9M82lNOCGv16/branch/paLe7CSyuYNsH6s5vlwPVs/Product%2FOnline?type=design&node-id=13847%3A94431&mode=design&t=Il1YhRhfwDY6cO1H-1)

![Screenshot 2024-04-12 at 6 06 14 PM](https://github.com/GetJobber/atlantis/assets/104797704/fef7f877-f0ac-4576-ad83-26ebdc85930e)